migrator/migrations/m_208_to_m_209_policy_updates_for_4_6/conversion/utils.go

```diff
@@ -0,0 +1,43 @@
+package conversion
```

Comment:
Have we been pulling these packages in other migrations?  I didn't think we were but I could be wrong.  So I'm wondering if this is necessary

Reply:
I haven't found those in the "main" code, I think they only exists in migrations. I copied these files from some previous migration but just put it into a separate folder for clarity.